From 669ada497f5386ff97d45d2269217422f04cca5e Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Mon, 8 Oct 2018 13:33:11 -0400
Subject: [PATCH 031/103] initramfs: allow an optional wrapper script around
 initramfs generation

It is sometimes useful to have a secondary script call the
gen_initramfs_list.sh script. This allows special interpretation
of the initramfs root filesystem or other manipulations to be
injected.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 usr/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/Makefile b/usr/Makefile
index c12e6b15ce72..ed29c43088aa 100644
--- a/usr/Makefile
+++ b/usr/Makefile
@@ -55,7 +55,7 @@ cpio-data := $(obj)/initramfs_data.cpio
 hostprogs := gen_init_cpio
 
 # .initramfs_data.cpio.d is used to identify all files included
-# in initramfs and to detect if any files are added/removed.
+# in initramfs $(INITRAMFS_WRAPPER) and to detect if any files are added/removed.
 # Removed files are identified by directory timestamp being updated
 # The dependency list is generated by gen_initramfs.sh -l
 -include $(obj)/.initramfs_data.cpio.d
-- 
2.17.1

